src/App.js

```diff
@@ -1,28 +1,55 @@
 import './App.css'
-import Oggetti from './Components/OggettiComponents/Oggetti'
-import AllCategories from './Components/Filter/AllCategories'
-import Cart from './Components/Cart/Cart'
+import React, { useState, useEffect } from 'react'
+
 import Header from './Components/HeaderContent'
+import Shopping from './pages/Shopping'
+import Login from './pages/Login'
+import Register from './pages/Register'
+import Logout from './pages/Logout'
+
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom'
+import { checkShowMe } from './utils/api'
 
 function App() {
-  return (
-    <div>
-      <header>
-        <nav className='header_content'>
-          <Header />
-        </nav>
-      </header>
+  const [user, setUser] = useState(undefined);
 
-      <div className='aside'>
-        <AllCategories />
-        <Cart />
-      </div>
+  useEffect(() => {
+    async function getLoggedInUser() {
+      const user = await checkShowMe()
+      if (user) {
+        setUser(user)
+      }
+    }
+    getLoggedInUser()
```

Comment:
we have to handle async functions this way when running them inside a useEffect: https://ultimatecourses.com/blog/using-async-await-inside-react-use-effect-hook